Source/FakeItEasy.Net35/Compatibility.cs

```diff
@@ -100,13 +79,14 @@ public Tuple(T1 item1, T2 item2)
     }
 }
 
-namespace System.Linq
+namespace FakeItEasy
 {
+    using System;
     using System.Collections.Generic;
     using System.Diagnostics.CodeAnalysis;
 
     [SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1402:FileMayOnlyContainASingleClass", Justification = "Tidier.")]
-    public static class EnumerableExtensions
+    internal static partial class EnumerableExtensions

```

Comment:
Internal to agree with the rest of the class, now that they have the same namespace. In real life, I'd look a little closer to see if the class should be combined into one file or maybe whether we actually have 2 classes here.